{"uuid": "12a29544-7a0b-42b2-b123-1c160a87c73c", "children": ["a917f7ce-7db5-4d78-9d21-e84ca578c2f6"], "befores": [{"name": "xunit_setup_class_fixture_TestMpLogin", "status": "passed", "start": 1609742750367, "stop": 1609742754459}], "afters": [{"name": "xunit_setup_class_fixture_TestMpLogin::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\_pytest\\python.py\", line 815, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\_pytest\\python.py\", line 739, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\PyCharm 2019\\PycharmProjects\\uiAutoTestHmtt\\scripts\\test01_mp_login.py\", line 20, in teardown_class\n    GetDriver.quit_web_driver()\n  File \"D:\\PyCharm 2019\\PycharmProjects\\uiAutoTestHmtt\\tools\\get_driver.py\", line 28, in quit_web_driver\n    cls.__web_driver.quit()\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 153, in quit\n    RemoteWebDriver.quit(self)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\request.py\", line 75, in request\n    method, url, fields=fields, headers=headers, **urlopen_kw\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\request.py\", line 96, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 796, in urlopen\n    **response_kw\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 706, in urlopen\n    chunked=chunked,\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connectionpool.py\", line 394, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connection.py\", line 234, in request\n    super(HTTPConnection, self).request(method, url, body=body, headers=headers)\n  File \"D:\\Python3.7\\lib\\http\\client.py\", line 1229, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"D:\\Python3.7\\lib\\http\\client.py\", line 1275, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"D:\\Python3.7\\lib\\http\\client.py\", line 1224, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"D:\\Python3.7\\lib\\http\\client.py\", line 1016, in _send_output\n    self.send(msg)\n  File \"D:\\Python3.7\\lib\\http\\client.py\", line 956, in send\n    self.connect()\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connection.py\", line 200, in connect\n    conn = self._new_conn()\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\connection.py\", line 170, in _new_conn\n    (self._dns_host, self.port), self.timeout, **extra_kw\n  File \"d:\\pycharm 2019\\pycharmprojects\\uiautotesthmtt\\venv\\lib\\site-packages\\urllib3\\util\\connection.py\", line 86, in create_connection\n    sock.connect(sa)\n"}, "start": 1609742779166, "stop": 1609742783213}], "start": 1609742750367, "stop": 1609742783213}